answers.md

```diff
@@ -1 +1,18 @@
 # Answers Go In This File
+//Shaina 
+//Qustion 1
+1. Git is a tool used by software developers to manage their code. Git keeps track of code by saving existing code. Software developers use it to code with others by branching, merging commits, and adding their code to the main base. Git is a local repository.
+//Question 2
+2. GitHub is like a social media platform, or a version control system, for coders or software developers. It allows software developers to start repositories, fork other repositories, and request pulls. GitHub is a remote repository.
+//Question 3
+3. Commit is a separate branch of a repository. It allows developers to change or add to the main code without changing or affecting the original or main base. Editing code in a commit is different from editing a document or file in a way that if you make a change in a google document or file, it will affect the document or file. Everyone will see the change you made to the original document or file. Commit allows you to make changes you like without affecting the original repository. If you want someone to see your commit and make changes to the original repository, you can request a pull to merge your commit and add it to the original repository. 
+//Question 4
+4. If you want to command git to add a file to a repository, type in git 'git add file.txt' or 'git add file.md'. If you want to add a directory to a repository, type, 'git add directory/'. If you like to add both directory and file, type 'git add .'. The period after 'git add' allows you to add everything to the repository. 
+//Question 5
+5. If you like to upload your local repository from git to GitHub, your remote repository, you’d like to command the computer by typing, “git push”. This allows you to transfer your commit to a remote repository. 
```

Comment:
This is great! Just noting that you can wrap "git push" in backticks (see the top left part of your keyboard) to make the command look like code. For example:

```
`git push`
```
Will look like this: `git push`

Reply:
Note: this isn't required but is a nice bit of markdown know-how